ui/objects/semantic-ui/js/objects/editform/components/RestorationWork.jsx

```diff
@@ -70,7 +70,7 @@ export const RestorationWork = ({
                   toolbar:
                     "bold italic | bullist numlist | outdent indent | undo redo",
                   valid_elements:
-                    "a,abbr,acronym,b,blockquote,br,code,div,table,tbody,td,th,tr,em,h1,h2,h3,h4,h5,i,li,ol,p,pre,span,strike,strong,sub,sup,u,ul",
+                    "a[href|title|name|target|rel],abbr[title],acronym[title],b,blockquote,br,code,div,table,tbody,td,th,tr,em,h1,h2,h3,h4,h5,i,li,ol,p,pre,span,strike,strong,sub,sup,u,ul",
```

Comment:
Same thing as https://github.com/Narodni-repozitar/nr-docs/pull/117#discussion_r1634395472 and https://github.com/VSCHT/restaurovani/pull/65#discussion_r1636220137